README.md

```diff
@@ -1,87 +1,77 @@
 
-### Irma 1
-`source /lupus/ngi/conf/sourceme_<SITE>.sh` where `<site>` is `upps` for `funk_004`, and `sthlm` for `funk_006` respectively.
+Manually place any additional files that need to be synced over under `/lupus/ngi/`
 
-`source activate NGI` to start the enviroment.
+Run `python sync.py <remote dest>` to rsync all files under `/lupus/ngi/` from irma3 to irma1. If no directory is given the default is `/lupus/ngi/`
 
-The run software in accordance with how it is typically is used.
+###Manual initiations on irma1
 
-#### Single use commands
-`crontab /lupus/ngi/conf/crontab_<site>` (user) initializes the first instance of cron for the user. No posterior loading is required.
+Run `crontab /lupus/ngi/conf/crontab_<site>` once per user to initialize the first instance of cron for the user. No posterior loading is required.
 
-`/lupus/ngi/resources/create_ngi_pipeline_dirs.sh <project_name>` (project) creates the log and db directories for <project_name>.
+Run `/lupus/ngi/resources/create_ngi_pipeline_dirs.sh <project_name>` once per project (i.e. ngi2016003) to create the log and db directories for NGI pipeline.
 
-`/lupus/ngi/sw/piper/gen_GATK_ref.sh` (global) generates the required GATK indexes to run piper.
+Run `/lupus/ngi/sw/piper/gen_GATK_ref.sh` one time ever to generate the required GATK indexes to run piper.
 
-## Pipeline integrity verification
+###Quick integrity verification
 
-`source /lupus/ngi/conf/sourceme_<SITE>.sh` where `<site>` is `upps` for `funk_004`, and `sthlm` for `funk_006` respectively.
+Run `source /lupus/ngi/conf/sourceme_<SITE>.sh` where <site> is upps to initialize variables funk_004, and sthlm to initialize funk_006 variables respectively.
 
-`source activate NGI` to start the enviroment.
+Run source activate NGI to start the enviroment.
 
 `python NGI_pipeline_test.py create --fastq1 <R1.fastq.gz> --fastq2 <R2.fastq.gz> --FC 1` creates a simulated flowcell.
 
 Run `ngi_pipeline_start.py` with the commands `organize flowcell`, `analyze project` and `qc project` to organize, analyze and qc the generated data respectively.
 
-## Pipeline run-time dependecies
-
-- A working connection to `charon` or `charon-dev` as well as a generated API token. 
-
-- A valid GATK key placed under `/lupus/ngi/irma3/deploy/files`. The filename must be specified in the `gatk_key` variable in `host_vars/127.0.0.1/main.yml`. 
-
-- A `charon_credentials.yml` file placed under `/lupus/ngi/irma3/host_vars/127.0.0.1/` to set the variables `charon_base_url`, `charon_api_token_upps` and `charon_api_token_sthlm`.
-
-## File details
+###Other worthwhile information
 
-Global configuration values set in `host_vars/127.0.0.1/main.yml`, and each respective role's in the `<role>/defaults/main.yml` file. 
+Deploying to sw requires the deployer to be in both the `ngi-sw` and the `ngi` groups. Everything under `/lupus/ngi/` is owned by `ngi-sw`, except for the `/db/` and `/log/` directories which are owned by `ngi`.
```

Comment:
`db` and `logs` are not present any longer. (They live under respective project's dirs)